src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/DropCommand.java

```diff
@@ -0,0 +1,88 @@
+package io.github.thebusybiscuit.slimefun4.core.commands.subcommands;
+
+import javax.annotation.Nonnull;
+import java.util.Locale;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import io.github.thebusybiscuit.cscorelib2.item.CustomItem;
+import io.github.thebusybiscuit.slimefun4.core.commands.SlimefunCommand;
+import io.github.thebusybiscuit.slimefun4.core.commands.SubCommand;
+import io.github.thebusybiscuit.slimefun4.core.multiblocks.MultiBlockMachine;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.utils.PatternUtils;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+
+/**
+ * The {@link DropCommand} drops a {@link SlimefunItem} of
+ * a specified quantity at the specified coordinates.
+ *
+ * @author NCBPFluffyBear
+ * @author TheBusyBiscuit (Based off of GiveCommand)
+ */
+class DropCommand extends SubCommand {
+
+    private static final String PLACEHOLDER_WORLD = "%world%";
+    private static final String PLACEHOLDER_X = "%x%";
+    private static final String PLACEHOLDER_Y = "%y%";
+    private static final String PLACEHOLDER_Z = "%z%";
+    private static final String PLACEHOLDER_ITEM = "%item%";
+    private static final String PLACEHOLDER_AMOUNT = "%amount%";
```

Comment:
Always hated %% as a placeholder prefix/suffix, I prefer {} but that's my preference. {world} just looks better and for me is more readable than %world%.

Others opinions?